__tests__/app.test.js

```diff
@@ -76,3 +160,58 @@ describe('GET/api/articles/:article_id', () => {
   });
 });
 
+describe('GET/api/articles/:article_id/comments', () => {
+  it('GET:200 and should respond with an an array of comments for the given article_id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(11);
+        console.log(comments[0]);
+        console.log(comments);
+        expect(comments[1]).toMatchObject({
+          comment_id: 2,
+          votes: 14,
+          author: 'butter_bridge',
+          body: 'The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.',
+          article_id: 1,
+        })
+        comments.forEach((comment) => {
+          expect (comment.hasOwnProperty('comment_id')).toBe(true);
```

Comment:
same comment as above, please add in checks for the typeof the property values coming back